lib/github_changelog_generator/project_name_finder.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+module GitHubChangelogGenerator
+  class ProjectNameFinder
```

Comment:
I find that trying to write a class level comment often helps with finding a fitting name. What purpose does this class serve, how does one use it?